docs/topics/jvm/jvm-create-project-with-spring-boot.md

```diff
@@ -153,7 +151,7 @@ fun main(args: Array<String>) {
    </def>
    <def title="Program entry point – main()">
       <p>The <a href="basic-syntax.md#program-entry-point"><code>main()</code></a> function is the entry point to the application.</p>
-      <p>It is declared as a <a href="functions.md#function-scope">top-level function</a> outside the <code>DemoApplication</code> class. The <code>main()</code> function invokes the Spring's <code>runApplication(&amp;args)</code> function to start the application with the Spring Framework.</p>
+      <p>It is declared as a <a href="functions.md#function-scope">top-level function</a> outside the <code>DemoApplication</code> class. The <code>main()</code> function invokes the Spring's <code>runApplication(*args)</code> function to start the application with the Spring Framework.</p>
```

Comment:
```suggestion
      <p>It is declared as a <a href="functions.md#function-scope">top-level function</a> outside the <code>DemoApplication</code> class. The <code>main()</code> function invokes the Spring's <code>runApplication(&amp;args)</code> function to start the application with the Spring Framework.</p>
```

Reply:
That is an escaping of the asterisk character.

Reply:
Oh, I see.
Sorry, I'll create a separate commit with the fix.
It seems that I need a vacation after 1.9.0 release :D

Reply:
That must be tough. Get a nice rest and congratulations for the 1.9.0 release!